ground/gcs/src/plugins/uavsettingsimportexport/importsummary.cpp

```diff
@@ -2,6 +2,7 @@
  ******************************************************************************
  *
  * @file       importsummary.cpp
+ * @author     dRonin, http://dRonin.org/, Copyright (C) 2015
```

Comment:
can you move it after TL, also since you made some changes might as well
extend TL to 2016